my-app/src/components/userCardSection/user-card-section.component.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./user-card-section.css";
+
+const UserCardSection = ({ sections }) => {
```

Comment:
I'd suggest having 2 parameters: title and text, instead of the single "sections" - that way the expectation is really clear what the component needs as parameters, while with section, I need to check the code below to see what structure is needed.